try {
    hello();
} catch (error) {
    console.log('ERROR')
}